app/views/councillor_contributions/_contribution_form_input.html.haml

```diff
@@ -0,0 +1,15 @@
+%fieldset
+  %legend
+    Add a councillor
+    = f.fields_for :suggested_councillors, @councillor_contribution.suggested_councillors.last do |suggested_councillor_field|
+      %p{:class => "nameField"}
+        = suggested_councillor_field.label :name
+        = suggested_councillor_field.text_field :name
+      %p{:class => "emailField"}
+        = suggested_councillor_field.label :email
+        = suggested_councillor_field.text_field :email
+  %p.button{:id => "addMoreCouncillor"}
```

Comment:
@hisayohorie what do you recon about moving the buttons out of the partial into app/views/councillor_contributions/new.html.haml ? That way this partial keeps a tight focus on just the HTML for inputing a councillor, and the buttons for other actions stay separate.